packages/core/components/DraggableComponent/styles.module.css

```diff
@@ -99,13 +99,12 @@
 
 .DraggableComponent-actions {
   position: absolute;
-  right: 6.5px;
+  right: -1px;
   width: auto;
-  top: -48px;
+  top: -37px;
```

Comment:
I don't think moving the toolbar here is the right solution, as leaving space between the toolbar and the border is a deliberate design choice that gives. I agree the flashing is annoying, but I think the better solution would be to create an invisible spacer between them that prevents the hover from bubbling up.

Reply:
Yup happy to implement that approach instead.

Reply:
fixed